tests/math/test_functions.py

```diff
@@ -1044,6 +1048,37 @@ def expected_grad(weights):
             -np.cos(a) * np.cos(c) * np.sin(b) ** 2
         ])
 
+    def test_weird_wires(self, in_tape_mode, tol):

```

Comment:
lol. `custom`?

Reply:
hahaha I believe @johannesjmeyer has a test called `test_weird_wires` in the circuit drawer, so I stole the name from there

Reply:
or maybe I'm imagining it and made it up

Reply:
Sounds plausible though